### DISCUSSION TITLE
Copy settings from another device

### DISCUSSION THREAD

**Discussion by robsonvitor** [netbox-community/netbox#5810] [General]
Hi,

I searched, but I didn't find it in the application.

Is it possible to copy the settings from one device to another? 

Example: a switch has stopped working. I need to keep it registered in Netbox, however, I want to copy its interface settings to another new switch. 

Is this possible in the current situation of Netbox, or is it an idea to be implemented? 

Thanks

**Comment by candlerb** [comment]
There's a related discussion here (about moving cables): https://groups.google.com/g/netbox-discuss/c/E2bm97chQRE

If you are doing a device swap-out, then it's simpler just to modify the Device in Netbox with the new serial number / asset tag and MAC addresses (if any).

Netbox isn't really an asset management or lifecycle system; it just reflects how the network should look *now*.  If you want to record things like when you bought something, when you sent it off for repair, and which spares you have on hand, then you may be better combining it with a separate asset management system.

**Reply by jpobeda** [reply]
I had this case last week and went down that path. I gave it some thought and came to that conclusion as well. Don't over complicate yourself :D and the end that switch will either trigger RMA or to be written off.

**Reply by robsonvitor** [reply]
I understand this definition of Netbox. I believe that this could be considered, as sometimes we need to maintain the equipment's life cycle. It is only a suggestion. I see a lot of potential in Netbox and I hope to contribute more soon.

Sorry for my English, maybe the translation was not good.
